content/300-guides/050-database/880-troubleshooting-orm/100-help-articles/400-nextjs-prisma-client-dev-practices.mdx

```diff
@@ -23,19 +23,15 @@ The solution in this case is to instantiate a single instance `PrismaClient` and
 ```ts file=./db
 import { PrismaClient } from '@prisma/client'
 
-declare global {
```

Comment:
I agree that this might not be necessary if `prisma` as a global isn't useful. Looking at the example below, it looks not so useful indeed since we're using imports.